function-app-deploy/README.md

```diff
@@ -0,0 +1,26 @@
+# GitHub Action for deploying to Azure Function App
+
+
+## Usage
+
+```
+
+action "Deploy to Azure  Function App" {
+  uses = "Azure/github-actions/functions@master"
+  needs = ["Azure Login"]
+  env = {
+    AZURE_APP_NAME = "<Azure App Name>"
+    AZURE_APP_PACKAGE_LOCATION = "<Relative path in your repository>"
```

Comment:
relative path to what ? it can be application folder or can it be .zip ?

Reply:
@vincentdass  can you suggest a description we can use here ?

Reply:
done